routing/router.go

```diff
@@ -480,6 +481,7 @@ func (r *ChannelRouter) Start() error {
 	if err != nil {
 		return err
 	}
+	r.bestHeight = uint32(bestHeight)
```

Comment:
nit: while we're at it, shouldn't we also use `atomic.Store...` here? Just for consistency's sake.

Reply:
Happy to do it, but I am not sure if it is really better. Typically thread-safe objects have initialization code that isn't thread-safe because it is supposed to be called only once. For example `sync.Mutex` locks are often not obtained either.

Reply:
Hmm, makes sense. Okay, let's leave it the way it is now then.